a_star_algo.py

```diff
@@ -0,0 +1,90 @@
+tags = {
```

Comment:
- [ ] @karthmnz Add/Use 10 emergency dispatch points as hard-coded list for easy testing